src/Views/HardwareView.vala

```diff
@@ -590,7 +590,11 @@ public class About.HardwareView : Gtk.Grid {
     private string custom_format_size (uint64 size, bool iec_unit) {
         uint divisor = iec_unit ? 1024 : 1000;
 
+#if HAS_GLIB_2_73
         const string[] UNITS = {"kB", "MB", "GB", "TB", "PB", "EB"};
+#else
+        const string[] UNITS = {"%.1f kB", "%.1f MB", "%.1f GB", "%.1f TB", "%.1f PB", "%.1f EB"};
```

Comment:
@tintou  @ryonakano  looks like this regresses the number format to show float instead of int? So it's showing `3.0 GB` instead of expected rounded `3 GB`

Reply:
Unfortunately there is nothing we can do until we have a newer GLib and it's not a big deal even for casual users :wink: